advanced/creating-an-icon-pack.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: default
+title: Create an icon pack for TinyMCE
+title_nav: Create an icon pack
+description_short: Introducing icon pack creation.
+description: How to make your own icon pack.
+keywords: create creator skin icon
+---
+
+TinyMCE 5 introduced icon packs for customizing the editor icons.
+
+### Prerequisites
+
+This guide assumes:
+
+* Familiarity with the command line and running commands.
+* [NodeJS](https://nodejs.org/en/), [NPM](https://www.npmjs.com), and [Gulp](https://gulpjs.com/) are already installed.
+* Optional: [`git`](https://git-scm.com/) is already installed.
+
+## How icons work in TinyMCE
+
+A TinyMCE icon pack is a `.js` file containing strings of [SVG's](https://developer.mozilla.org/en-US/docs/Web/SVG). An icon pack can be used: to include one or more custom icons; or to replace some or all of the default TinyMCE icons.
+
+An icon pack only requires the custom icons to be included; the default TinyMCE icons are used as a fallback for icons missing from the custom icon pack.
+
+> Don't forget to explore our ready-to-use Premium Icon Packs such as 'Material' icons, and a smaller version of our default icons at [Tiny Skins and Icon Packs](https://apps.tiny.cloud/products/skins-and-icon-packs/).
+
+## Creating a TinyMCE icon pack
+
+To create a custom icon pack:
+* [Download and setup the icon pack template](#downloadandsetuptheiconpacktemplate)
+* [Add the SVG files](#addthesvgfiles)
+* [Build the icon pack](#buildtheiconpack)
+
+### Download and setup the icon pack template
+
+To use the TinyMCE icon pack template project:
+
+1. Download the [TinyMCE Oxide Icon Pack Template](https://github.com/tinymce/oxide-icon-pack-template) by either:
+
+    - Downloading the `.zip` file from the [Oxide Icon Pack Template GitHub page](https://github.com/tinymce/oxide-icon-pack-template) and extract the contents.
+    - From a terminal or command prompt, use git to clone the GitHub repository, such as:
+
+        ```sh
+        $ git clone https://github.com/tinymce/oxide-icon-pack-template.git
```

Comment:
Not sure if you're worried, but the indentation here and in a few other places looks a little bit weird when rendered as it causes a large left padding.

Reply:
wasn't sure if the build software would be ok with me removing the indent, apparently it does, so fixed.